packages/ui/stories/proposal/ProposalStatus.stories.tsx

```diff
@@ -7,12 +7,15 @@ export default {
   component: ProposalStatus,
 } as ComponentMeta<typeof ProposalStatus>
 
-const Template: ComponentStory<typeof ProposalStatus> = (args) => (
-  <ProposalStatus {...args} />
+const Template: ComponentStory<typeof ProposalStatus> = () => (
+  <div className="flex flex-row gap-12 items-center">
+    <ProposalStatus status="open" />
+    <ProposalStatus status="rejected" />
+    <ProposalStatus status="approved" />
```

Comment:
Any way we can use our enum for these? Conversely, any reason we might not want to use our enum for these?

Reply:
This is a bit more readable, and the typescript `${enum}` type I used in the component lets you pass the enum or the value.

Reply:
Got it. Why not add a comment that says this? Good to explain what's going to use that enum and how - in this case, the typechecker's use of that enum is implicit, but a comment would make it explicit.

Reply:
I'd rather use the enum than add a comment, so I'll change it!